app/src/main/res/values/strings.xml

```diff
@@ -44,6 +44,8 @@
   <string name="status_location_update">Location update</string>
   <string name="status_network_online">Network online</string>
   <string name="status_network_offline">Network offline</string>
+  <string name="status_widget_description">The widget shows the service status</string>
```

Comment:
Can we rename these to have `widget` prefix and move to the bottom? I don't want to mix them up with other stuff.

Reply:
Should I rename these strings to `widget_description` and `widget_label`?

Reply:
Yes, of course.